src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -0,0 +1,829 @@
+<template>
+  <div
+    :id="props.id"
+    ref="codeBlock"
+    class="k-code-block"
+    :class="{ 'k-code-block--has-line-numbers': props.showLineNumbers }"
```

Comment:
We do not use BEM CSS (or other `--` syntax). Please just use a single `-` in class names; this applies for the entire file.

Reply:
I don’t feel particularly strongly about this so I’m changing it.

That being said, I don’t think like project should prohibit the use of BEM-style selectors just because they look like BEM-style selectors. It’s of no negative consequence to anyone making use of them because they’re in no way special while they may aid those used to and familiar to BEM-style selectors in identifying intent without referring to markup (while reading styles) or styles (while reading markup).

In any case, as I wrote above, I’m changing it, so consider this merely a comment.

Reply:
Appreciate the change. It's more about consistency. I'm working to clean up a lot of the inconsistencies in some of our repositories and having different "flavors" of CSS within the projects isn't ideal.

Another example is `camelCase` versus `kebab-case` (preferred) selectors.